File: Anul II/Medii de proiectare si programare/Laborator/lab08/C#/Ubb.BikeContest.Networking/Protocol/Response/AllTeamsResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ubb.BikeContest.Model;

namespace Ubb.BikeContest.Networking.Protocol.Response
{
    [Serializable]
    public class AllTeamsResponse : IResponse
    {
        private readonly List<Team> teams;

        public AllTeamsResponse(List<Team> teams)
        {
            this.teams = teams;
        }

        public List<Team> Teams { get { return teams; } }
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab10/networking/src/main/java/com/ubb/protocol/response/AllTeamsResponse.java
package com.ubb.protocol.response;

import com.ubb.model.Team;

import java.util.Collection;

public class AllTeamsResponse implements Response {
    private final Collection<Team> teams;

    public AllTeamsResponse(Collection<Team> teams) {
        this.teams = teams;
    }

    public Collection<Team> getTeams() {
        return teams;
    }
}
